projects/demo-integrations/src/tests/component-testing/predicate/predicate.cy.ts

```diff
@@ -0,0 +1,46 @@
+import {MaskitoOptions} from '@maskito/core';
+
+import {TestInput} from '../utils';
+import {SynchronousTestInputComponent} from './synchronous-test-input.component';
+
+describe('@maskito/angular | Predicate', () => {
+    const cardMask: MaskitoOptions = {
```

Comment:
Is it used only inside `it('supports asynchronous predicate', ...)` ?

Let's move it inside this spec.

Reply:
Moved